core/deployment/src/main/java/io/quarkus/deployment/steps/NativeImageAutoFeatureStep.java

```diff
@@ -201,6 +173,21 @@ public void write(String s, byte[] bytes) {
             }
         }
 
+        /* Resource includes and excludes */
+        if (!resourcePatterns.isEmpty()) {
+            ResultHandle resourcesRegistrySingleton = overallCatch.invokeStaticMethod(IMAGE_SINGLETONS_LOOKUP,
+                    overallCatch.loadClass("com.oracle.svm.core.configure.ResourcesRegistry"));
+            TryBlock tc = overallCatch.tryBlock();
+            for (NativeImageResourcePatternsBuildItem resourcePatternsItem : resourcePatterns) {
+                for (String pattern : resourcePatternsItem.getIncludePatterns()) {
+                    tc.invokeInterfaceMethod(RESOURCES_REGISTRY_ADD_RESOURCES, resourcesRegistrySingleton,
+                            overallCatch.load(pattern));
```

Comment:
you should use `tc` here to load the pattern.

Reply:
Thanks, let me have a look.

Reply:
fixed in 8f5c2b5